uui-components/src/buttons/Button.tsx

```diff
@@ -1,10 +1,35 @@
 import * as React from 'react';
-import { Icon, uuiElement, uuiMarkers, CX, cx, IHasIcon, IDropdownToggler, IHasCaption, devLogger } from '@epam/uui-core';
-import { Clickable, ClickableComponentProps, UnionRawProps } from '../widgets';
+import {
+    Icon,
+    uuiElement,
+    uuiMarkers,
+    CX,
+    cx,
+    IHasIcon,
+    IDropdownToggler,
+    IHasCaption,
+    devLogger,
+    IClickable, IAnalyticableClick, IHasTabIndex, IDisableable, IHasCX, ICanRedirect,
+} from '@epam/uui-core';
+import {
+    AnchorNavigationProps,
+    ButtonNavigationProps,
+    Clickable,
+    HrefNavigationProps,
+    LinkButtonNavigationProps,
+} from '../widgets';
 import { IconContainer } from '../layout';
 import css from './Button.module.scss';
 
-export type ButtonProps = ClickableComponentProps & IDropdownToggler & IHasIcon & IHasCaption & {
+export type UnionButtonNavigationProps = HrefNavigationProps | LinkButtonNavigationProps | ButtonNavigationProps | AnchorNavigationProps;
+
+export type ButtonRawProps =
```

Comment:
Can we use IHasRawProps<(React.AnchorHTMLAttributes<HTMLAnchorElement> | React.ButtonHTMLAttributes<HTMLButtonElement>> instead  of re-declaring the whole type here?

Reply:
I will try to change